data/json/vehicleparts/vehicle_parts.json

```diff
@@ -602,8 +602,8 @@
     "damage_modifier": 80,
     "durability": 100,
     "description": "A small fridge.  When turned on, it will cool the food inside, extending the time until the food spoils.",
-    "//": "Use average consumption, not the max on the appliance rating plate. 30W ~ 260kWh per annum",
-    "epower": "-30 W",
+    "//": "Use average consumption, not the max on the appliance rating plate. For big fridge 30W ~ 260kWh per annum is good, but for mini variant with decrased volume by ~ 6 time, surface area ~ 3-4 times and insulation thickness only by 2 time 5W is quite good ",
```

Comment:
We don't need to do the math you did to determine this, we can just look at similar minifridges (75L ~= 2.6 cubic ft):
https://www.amazon.com/Frestec-Fridge-FR-250-WH/dp/B0BBVRMWWP
https://www.amazon.com/Upstreman-Refrigerator-Adjustable-Thermostat-Red-CR25/dp/B0BVLPFWWP
Both of these list an energy consumption of 210kWh per annum, so that is... [24W](https://www.wolframalpha.com/input?i=210kWh+%2F+%28year+in+seconds%29).

Reply:
If big fridge comsumption has been looked for many time
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/102726167/156a0788-b07e-4de1-8fcc-e8d0ecf69bbb)
So, **everage** consumption cannot be decreased only by 20%, when you decrasing volume by ~ 6 time, surface area ~ 3-4 times and insulation thickness only by 2 time. At least energy consumtion in this case should be decresed by 4-5 times
I also found lots of different models, like https://www.amazon.com/DOMETIC-75-Liter-Portable-Refrigerator-Freezer/dp/B09FS4YJ8F which is Refrigerator+freezer and consuming ultra low. If I am right, it have ~ 86 kWH per annum, what is 10W. And our ingame mini fridges have good insulation as I can understand. Thicker insulation - less energy wasting. 
If the ingame fridge looks like cheap fridge from amazon - it doesnt mean, that it will work like that

Its hard to find good examples, but if you check oficcial ENERGYSTAR site, you can see, that there is refrigerators, which can hold more litres and comsuming less, than linked by you from amazon (if you make sorting by annual energy use)
https://www.energystar.gov/productfinder/product/certified-residential-refrigerators/

Reply:
I'm not sure where the numbers come from for the amazon link, the only one I see there is "0.01 Kilowatt Hours" which doesn't make much sense to me. Past that, I personally would not say that is a minifridge.
Looking at the 49 2.6 cubic feet (~75L) models on the energystar website, I find the following frequency for energy consumption values (in kWh):
```
      1 208
      7 214
     36 215
      2 216
      1 228
      1 253
      1 259
```
I don't really think it's worth the time to think about a hypothetical fridge, when we have all this data for fridges that match this one.

Reply:
Its worth the time. Because in game we have tech specs of fridge (volume, recipe parts and energy comsumption) and can calculate , when at sites lots of this parameters will be very different or hidden
About energy star, if the sorting by annual energy use is enable, so


<details>
  <summary>Spoiler warning</summary>
  
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/102726167/d8d06bc1-8e28-40ff-b8f7-d6b6b24b98c6)
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/102726167/57ed0d4f-a448-4768-849e-c7dbaa04d0b8)
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/102726167/131637c0-5291-4c7f-93e2-a3ed369ad90d)

</details>
Yes, they can hold more, but if the bigger fridge can do that, so do and less bigger

Insulation is playing very important role in energy consumption and cols saving
<details>
  <summary>Spoiler warning</summary>
  
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/102726167/67b49024-438c-4a28-9c66-dc051695a1c9)

</details>

We can make it that way, that the mini freezers crafted by player (and rarely finded in cars) have thick insulation and that why they need less energy, what is vital in the world without standart power generation (and even when you are traveling by car). And if somebody want to make cheap variant like the low price variants from amazon - he can do it. And the cheapest variant can be less bigger bacause of less insulation thickness. I also made the logic in that way, that have 1 big fridge(500L/30W)/freezer(450L/35W) is energy economical than have 6,6 minifridgres (~500L/33W) and 1 nig freezer (450L/35W) vs 6 minifreezers (450L/42W) , because of the calculation in the pr